dot/import_integration_test.go

```diff
@@ -0,0 +1,121 @@
+// Copyright 2021 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+//go:build integration
+// +build integration
+
+package dot
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"math/big"
+	"os"
+	"testing"
+
+	"github.com/ChainSafe/gossamer/dot/types"
+	"github.com/ChainSafe/gossamer/lib/common"
+	"github.com/ChainSafe/gossamer/lib/utils"
+	"github.com/ChainSafe/gossamer/pkg/scale"
+
+	"github.com/stretchr/testify/require"
+)
+
+func setupStateFile(t *testing.T) string {
+	const filename = "../lib/runtime/test_data/kusama/block1482002.out"
+
+	data, err := ioutil.ReadFile(filename)
+	require.NoError(t, err)
+
+	rpcPairs := make(map[string]interface{})
+	err = json.Unmarshal(data, &rpcPairs)
+	require.NoError(t, err)
+	pairs := rpcPairs["result"].([]interface{})
+
+	bz, err := json.Marshal(pairs)
+	require.NoError(t, err)
+
+	fp := "./test_data/state.json"
+	err = ioutil.WriteFile(fp, bz, 0777)
+	require.NoError(t, err)
```

Comment:
Write to a `t.TempDir()` instead.

```suggestion
	fp := filepath.Join(t.TempDir(), "state.json")
	err = ioutil.WriteFile(fp, bz, 0777)
	require.NoError(t, err)
```